### DISCUSSION TITLE
Troubles running  climatological (hourly averages) data from HYCOM

### DISCUSSION THREAD

**Discussion by N1coleR** [OpenDrift/opendrift#1234] [Q&A] [answered]
Dear community:

I have downloaded UV data from HYCOM for several months with their respective years and made an hourly climatological average using MatLab (a new .nc was created with the averaged data)

When I try to run the simulation in OpenDrift (PelagicEggDrift) with the climatological data I get this error:

ERROR opendrift.models.basemodel:1277: ufunc 'add' did not contain a loop with signature matching types (dtype('float32'), dtype('<U1')) -> None

I have tested running OpenDrift with the original HYCOM data and it works correctly. Everything indicates that there is something in my climatological file, however,  when I maked the climatological .nc I was careful to keep the data structure, metadata, and data type the same as the original HYCOM file.  I have checked it carefully and everything seems fine.

Does anyone have an idea why this error appears?
I attached to you a link to DropBox where you can find the files

https://www.dropbox.com/scl/fo/h2cqfcprwx7fkhv4ydam8/h?rlkey=9136va04b54tdsxulixxkrx9z&dl=0

Thank you in advanced!

**Comment by knutfrode** [comment, marked as answer]
Hi,
For `water_v`you have `standard_name = "eastward_sea_water_velocity"`, but correct should be `northward....`

**Reply by N1coleR** [reply]
Thank you for the observation!!
